src/main/java/shortestpath/pathfinder/CollisionMap.java

```diff
@@ -2,19 +2,21 @@
 
 import java.io.IOException;
 import java.io.UncheckedIOException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
It is a RuneLite convention to not use wild card imports, so I would prefer if you could list them as before.